### PR TITLE
tools: debug tools nav + telemetry viewer 

### PR DESCRIPTION
**Pull request by smoralesd**
#### Description of changes

This PR is a follow up of #2515, where the **Telemetry Viewer** was introduced.

Adding a left nav for the debug tools page, which contains 2 navs: stores & telemetry viewer.

Add actions, store and action creator and add them to the dependency tree from the debug tools init file.

01 - debug tools nav and telemetry viewer

#### Pull request checklist
- [ ] Addresses an existing issue: #0000
- [x] Ran `yarn fastpass`
- [x] Added/updated relevant unit test(s) (and ran `yarn test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). See `CONTRIBUTING.md`.
- [x] (UI changes only) Added screenshots/GIFs to description above
- [ ] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: src/tests/unit/tests/debug-tools/components/debug-tools-nav.test.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import { DebugToolsNav, DebugToolsNavProps } from 'debug-tools/components/debug-tools-nav';
+import { ToolsNavKey } from 'debug-tools/stores/debug-tools-nav-store';
+import { shallow } from 'enzyme';
+import { INavLink, Nav } from 'office-ui-fabric-react';
+import * as React from 'react';
+import { Mock, Times } from 'typemoq';
+
+describe('DebugToolsNav', () => {
+    it('renders', () => {
+        const props: DebugToolsNavProps = {
```
Comment (waabid) [inline src/tests/unit/tests/debug-tools/components/debug-tools-nav.test.tsx:14]: Feels like you can use a beforeEach and share the props.

File: src/tests/unit/tests/debug-tools/components/debug-tools-view.test.tsx [hunk 2]
```diff
@@ -9,6 +9,7 @@ import {
 import { shallow } from 'enzyme';
 import * as React from 'react';
 
+//
```
Comment (waabid) [inline src/tests/unit/tests/debug-tools/components/debug-tools-view.test.tsx:12]: Accidental?

File: src/debug-tools/stores/debug-tools-nav-store.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        return {
+            selectedTool: 'stores',
+        };
+    }
+
+    protected addActionListeners(): void {
+        this.toolsNavActions.setSelectedTool.addListener(this.onSetSelectedTool);
+    }
+    private onSetSelectedTool = (selectedTool: ToolsNavKey) => {
+        if (this.state.selectedTool !== selectedTool) {
```
Comment (waabid) [inline src/debug-tools/stores/debug-tools-nav-store.ts:29]: nit: maybe use a guard clause here instead?

File: src/tests/unit/tests/debug-tools/components/debug-tools-nav.test.tsx (hunk 1 above)
Comment (smoralesd) [reply src/tests/unit/tests/debug-tools/components/debug-tools-nav.test.tsx:14]: update

File: src/tests/unit/tests/debug-tools/components/debug-tools-view.test.tsx (hunk 2 above)
Comment (smoralesd) [reply src/tests/unit/tests/debug-tools/components/debug-tools-view.test.tsx:12]: updated

File: src/debug-tools/stores/debug-tools-nav-store.ts (hunk 3 above)
Comment (smoralesd) [reply src/debug-tools/stores/debug-tools-nav-store.ts:29]: updated

**Comment by smoralesd** [comment]
@waabid thanks for the review.

**Pull request merged**